developer-guides/rest-api/subscriptions/README.md

```diff
@@ -0,0 +1 @@
+# Subscriptions
```

Comment:
@rafaelks Could you write the description of this api? like in [this file](https://github.com/RocketChat/docs/blob/master/developer-guides/rest-api/miscellaneous/README.md) Thanks!

Reply:
This is not an actual API... can I remove the README file just to maintain the `subscriptions` folder?

Reply:
@rafaelks if you do that, when someone clicks on the folder, they will be redirected to the home page, do you plan to add more files to that folder? Maybe we can just put a list of what is inside of the folder

Reply:
Done! 👍